dashboard/config/environment.rb

```diff
@@ -17,4 +17,7 @@
 # and we won't be migrating it to other environments. However, it shouldn't belong in
 # the schema, and we don't want it to show up when generating the schema on the prod
 # database. This line will ignore that table.
-ActiveRecord::SchemaDumper.ignore_tables = ['overflow_activities']
+# It also will ignore tables starting with an underscore, which are temporarily created
+# when we use the gh-ost online migration tool to migrate large tables
+# without significant downtime.
+ActiveRecord::SchemaDumper.ignore_tables = ['overflow_activities', /^_/]
```

Comment:
My experience with regular expressions is limited.  Will this pattern match any table name that does not start with `_`?  Or does it match any table name that is a single character that is not `_`?   Do we need to extend this pattern to say that any number of valid characters can follow the character that is not an underscore?

Reply:
It'll match any table that _does_ start with an underscore -- you could play around with something like [this](https://regex101.com/) if you want to test things out further. A little more info in the PR description about how I tested this out.